search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -748,6 +748,15 @@
     :revision_date
     :result-format})
 
+(def ^:private valid-deleted-granules-search-params
+  "Valid parameters for deleted granules search"
+  #{:revision-date
+    :provider
+    :parent-collection-id
+    :revision_date
+    :parent_collection_id
+    :result-format})
```

Comment:
We only support JSON right now and for other endpoints we support the result-format through a URL extension or header, not a query parameter. I'm wondering why this is in the list?

Reply:
It seems to be added automatically to the params at some point, deleted-collections has this also even though it only accepts xml